cli/dcoscli/service/main.py

```diff
@@ -305,4 +306,8 @@ def _log_marathon(follow, lines, ssh_config_file):
 
     emitter.publish(DefaultError("Running `{}`".format(cmd)))
 
+    # In the tests, we SIGKILL this process group, and the output
+    # doesn't get flushed unless we explicitly do so here.
+    sys.stderr.flush()
```

Comment:
I don't like complicating an interface just to support a test.  And we'll have to pass this through 3-4 layers of abstraction in `emitting`.  Let me know what you want to do.